nats-subscriber/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8
```

Comment:
The other Linux for Health images are "based off of" a [base image](https://github.com/LinuxForHealth/images/blob/master/base/Dockerfile) which in turn extends the UBI 8 minimal image. This base image provides us with:
- a standard application "root" directory, /opt/lfh
- a standard, non-privileged "app" user, lfh

If this docker file could utilize that base image you could either install node with `microdnf` (UBI minimal image package manager) or do a multistage build. We can use other images as "recipes" to see how it could be crafted.

Reply:
That base image reduces the image size by 75% - awesome!